diags/diag_nvme.c

```diff
@@ -974,11 +988,6 @@ extern int open_nvme(char *dev_path) {
 static void print_usage(char *command) {
 	printf("Usage: %s [-h] [-d <file>] [-f <file>] [<nvme_devices>]\n"
```

Comment:
The print_usage still prints [-d <file>] [-f <file>] . Should those also be removed ?

Reply:
Yes, good catch. They were removed from the details but not the core usage on the first line of the message.